mysql-client-tests/dotnet/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Data;
+
+using MySql.Data;
+using MySql.Data.MySqlClient;
+
+class TestException : Exception
+{
+    public TestException(string message)
+    {
+        Console.WriteLine(message);
+    }
+}
+
+public class DoltSQL
+{
+    public static int Main(string[] args)
+    {
+        if (args.Length != 3)
+        {
+            Console.WriteLine("Must supply $USER $PORT $REPO arguments.");
+            return 1;
+        }
+
+        var user = args[0];
+        var port = args[1];
+        var db = args[2];
+
+        string connStr = $"server=127.0.0.1;user={user};database={db};port={port};CharSet=utf8;";
+        MySqlConnection conn = new MySqlConnection(connStr);
+
+        try
+        {
+            conn.Open();
+            SetupTest(conn);
+            QueryTest(conn);
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine(ex.ToString());
+        }
+
+        conn.Close();
+        return 0;
+    }
+
+    public static void SetupTest(MySqlConnection conn)
```

Comment:
Should we add the read queries here and examine the results?

Reply:
ie. https://github.com/liquidata-inc/dolt/blob/c28953970d9cbf91621f9bd9ee53eb31edfe7be2/mysql-client-tests/python/pymysql-test.py#L4

I'm less concerned about examining the results than I am running a couple more queries.

Reply:
I see. You're just missing the describe. This is probably fine. Up to you...